### PR TITLE
[Bug 1201560] Show Last Update date in Mobile view

### PR DESCRIPTION
**Pull request by safwanrahman**
@mythmon A small fix.
r?

### CODE REVIEW

File: kitsune/wiki/templates/wiki/mobile/document.html [hunk 1]
```diff
@@ -44,6 +44,9 @@
         {{ _('Share this article: {link}')|f(link=link|safe) }}
       </p>
     {% endif %}
+    <p>
+      Last Update: {{ latest_approved_revision.created.date() }}
```
Comment (rehandalal) [inline kitsune/wiki/templates/wiki/mobile/document.html]: you can just change this to:
```
document.current_revision.created.date()
```

and then undo all the changes to `wiki/views.py`

**Comment by safwanrahman** [comment]
Thanks @rehandalal. My think was bit critical though it can be done easily! :+1:

**Comment by safwanrahman** [comment]
This should fix! :+1: r?

**Comment by rehandalal** [comment]
code looks good, but looking at the page I think it should actually go under the list of people that helped write the article and it should be lighter in color `color: #888;`.

**Comment by safwanrahman** [comment]
@rehandalal r?

**Comment by rehandalal** [comment]
good spot on the l10n. this is great, thanks!

**Pull request merged**